code/_onclick/hud/blob_overmind.dm

```diff
@@ -151,18 +151,17 @@
 		var/mob/eye/blob/B = usr
 		B.relocate_core()
 
+/atom/movable/screen/blob_power_display
+	name = "blob power"
+	icon_state = "block"
+	screen_loc = ui_health
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	plane = ABOVE_HUD_PLANE
+
```

Comment:
why does this live here if it isn't used here? am I missing surrounding context that makes this right?

Reply:
It's just under all the other blob screen movables didnt think I should move it

Reply:
works for me